hosted_logos/cryptomatte.svg

```diff
@@ -1 +1,187 @@
.492 79.315,72.583 79.460 C 72.983 80.093,73.499 80.671,74.850 82.000 C 75.643 82.779,76.398 83.492,76.530 83.583 C 77.048 83.945,77.454 84.170,78.333 84.583 C 78.837 84.819,79.367 85.077,79.510 85.155 C 82.420 86.742,86.478 83.959,87.587 79.617 C 88.185 77.273,87.074 75.687,84.690 75.482 C 82.418 75.287,82.024 75.149,80.947 74.175 C 80.028 73.344,79.433 70.288,80.108 69.869 C 80.711 69.495,80.413 69.126,79.139 68.669 C 77.176 67.965,75.878 67.314,73.032 65.603 C 66.947 61.944,65.601 61.436,61.887 61.397 C 58.899 61.365,58.833 61.349,58.833 60.667 C 58.833 60.014,58.873 60.001,61.291 59.838 C 65.120 59.580,67.557 59.983,74.040 61.942 C 76.113 62.569,76.683 62.726,78.000 63.036 C 82.867 64.182,88.247 62.553,90.089 59.378 C 90.368 58.898,90.378 58.770,90.181 58.211 C 89.886 57.373,88.522 53.879,88.245 53.250 C 86.569 49.445,86.336 48.999,86.030 49.006 C 85.517 49.016,83.980 49.872,83.167 50.600 C 82.536 51.165,80.820 52.000,80.291 52.000 C 78.985 52.000,77.500 50.674,77.500 49.508 C 77.500 49.154,77.045 49.035,76.742 49.310 C 76.464 49.563,73.818 50.214,71.500 50.600 C 68.171 51.155,67.214 51.330,66.083 51.585 C 65.442 51.730,64.617 51.913,64.250 51.993 C 61.939 52.494,58.877 54.353,57.446 56.124 C 57.083 56.573,56.626 56.605,56.169 56.213 C 55.775 55.876,55.745 55.609,56.064 55.304 C 56.191 55.182,56.922 54.480,57.689 53.742 C 59.422 52.075,62.994 49.500,63.574 49.500 C 63.608 49.500,64.337 49.173,65.193 48.774 C 66.049 48.374,67.650 47.669,68.750 47.207 C 74.982 44.589,77.903 42.009,79.076 38.087 L 79.324 37.257 77.627 35.004 C 76.693 33.764,75.766 32.563,75.566 32.333 C 74.953 31.632,72.914 29.273,72.577 28.875 C 72.193 28.422,71.797 28.408,70.917 28.817 C 69.966 29.258,69.249 30.019,69.067 30.777 C 69.005 31.037,68.896 31.325,68.825 31.417 C 68.754 31.508,68.645 31.883,68.583 32.250 C 68.056 35.335,66.143 37.128,64.337 36.227 C 63.710 35.914,63.868 35.782,62.320 37.914 C 61.856 38.555,61.425 39.117,61.363 39.164 C 61.301 39.211,60.950 39.624,60.583 40.081 C 60.217 40.539,59.801 41.029,59.660 41.171 C 59.519 41.313,58.961 41.989,58.419 42.673 C 57.878 43.357,57.391 43.954,57.339 44.000 C 56.795 44.472,54.167 48.287,54.167 48.603 C 54.167 48.656,54.060 48.936,53.929 49.225 C 53.497 50.180,53.032 52.568,53.012 53.929 C 52.995 55.129,52.573 55.500,51.227 55.500 C 49.536 55.500,49.291 54.977,49.664 52.167 C 50.170 48.349,50.447 46.831,50.679 46.592 C 50.764 46.505,50.833 46.291,50.833 46.117 C 50.833 45.943,50.951 45.527,51.096 45.192 C 51.240 44.857,51.463 44.338,51.591 44.039 C 51.719 43.740,51.930 43.252,52.058 42.956 C 52.187 42.659,53.083 40.804,54.049 38.833 C 56.079 34.695,56.500 33.771,56.500 33.459 C 56.500 33.335,56.569 33.162,56.654 33.075 C 56.738 32.988,56.894 32.504,56.999 32.000 C 57.872 27.806,57.147 24.595,55.018 23.217 C 54.317 22.763,53.952 22.628,52.167 22.161 C 51.561 22.002,50.101 21.969,49.667 22.104 M122.333 40.083 C 122.333 40.278,122.111 40.333,121.333 40.333 C 120.556 40.333,120.333 40.389,120.333 40.583 C 120.333 40.769,1
```

Comment:
Didn't see the matching entry for this one in landscape.yml - assuming https://github.com/Psyop/Cryptomatte

Reply:
Nm - I see it was there before :-)